webview/src/experiments/components/table/styles.module.scss

```diff
@@ -336,6 +378,21 @@ $workspace-row-edge-margin: $edge-padding - $cell-padding;
           background-color: $row-bg-selected-color;
         }
       }
+
+      .indicatorCount {
+        background-color: $indicator-selected-badge-background;
+        color: $indicator-selected-badge-foreground;
+        border: 1px solid $indicator-selected-badge-border;
+      }
+
+      .starSwitch:not([aria-checked='true']) svg {
+        fill: $star-switch-selection-fill-color;
```

Comment:
With this update, the star and cursor are now more viewable on the light theme 🎉 

Current:

<img width="235" alt="image" src="https://user-images.githubusercontent.com/43496356/189001376-a2008973-1933-49c0-8132-63a1cafd0c57.png">

PR:

<img width="226" alt="image" src="https://user-images.githubusercontent.com/43496356/189001401-5c3d01ac-ca8b-4810-b2b5-f8e1133c4a74.png">

Reply:
This is very good !